src/Google.Storage.V1/StorageClient.ListObjects.cs

```diff
@@ -71,19 +60,9 @@ public partial class StorageClient
         /// <param name="options">The options for the operation. May be null, in which case
         /// defaults will be supplied.</param>
         /// <returns>A sequence of objects within the bucket.</returns>
-        public IEnumerable<Object> ListObjects(string bucket, string prefix, ListObjectsOptions options = null)
-        {
-            var initialRequest = CreateListObjectsRequest(bucket, prefix, options);
-            return s_objectPageStreamer.Fetch(initialRequest);
-        }
-
-        private ObjectsResource.ListRequest CreateListObjectsRequest(string bucket, string prefix, ListObjectsOptions options)
+        public virtual IEnumerable<Object> ListObjects(string bucket, string prefix, ListObjectsOptions options = null)
```

Comment:
Like ListBuckets - why does the async method include "All" in the name, but not the sync method?